processInput.py

```diff
@@ -0,0 +1,20 @@
+import receiver_calculation
+import sys
+
+def getSenderInput():
+  temp = []
+  soc = []
+  for i in sys.stdin:
+    list_temp,list_soc = formatInputStream(i)
```

Comment:
It's hard to understand the role of `list_temp`. What do you want readers to understand by seeing that variable? Name it like you would explain.